app/poros/movie_review.rb

```diff
@@ -0,0 +1,15 @@
+class MovieReview
+  attr_reader  :author,
+               :content,
+               :content_snippet
+
+  def initialize(review)
+    @author = review[:author]
+    @content = review[:content]
+    @content_snippet = format_snippet(review[:content])
+  end
+
+  def format_snippet(snippet)
```

Comment:
Beautiful user thinking - I like that you wanted to limit the size of the review and (eventually), the user can just click the review to see the whole thing :)